Source/Meadow.Contracts/Peripherals/ITouchScreen.cs

```diff
@@ -1,28 +1,27 @@
-﻿namespace Meadow.Hardware
+﻿namespace Meadow.Hardware;
+
+/// <summary>
+/// An event delegate for touchscreen events
+/// </summary>
+/// <param name="x">The horizontal position of the touch</param>
+/// <param name="y">the vertical position of the touch </param>
+public delegate void TouchEventHandler(int x, int y);
+
+/// <summary>
+/// Represents a touch screen device
+/// </summary>
+public interface ITouchScreen
 {
     /// <summary>
-    /// An event delegate for touchscreen events
+    /// Event fired when the touchscreen is pressed
```

Comment:
_raised_, not fired. i can't remember why that's that the language rec.

Reply:
Blame ChatGPT.